### DISCUSSION TITLE
SOLVED - Can´t revert back to marlin - LCD bugs

### DISCUSSION THREAD

**Discussion by lanotlab** [Desuuuu/DGUS-reloaded-Klipper#54] [General]
I've tried using klipper, it worked up to a point, then I tried to revert back to marlin, but the LCD firmware is now buggy

here's a video of what's happening when I turn the printer on:
https://www.youtube.com/shorts/ByqnGbgb_zQ

PS. If I flash your firmware back, it works fine with klipper.

**Comment by Desuuuu** [comment]
Can you try following the steps outline in my comment [here](https://github.com/Desuuuu/DGUS-reloaded-Klipper/discussions/36#discussioncomment-3213662)?

**Reply by lanotlab** [reply]
already tried that. no difference. I renamed the 13 and 14 files, it flashed, but the screen is still flickering.

**Reply by lanotlab** [reply]
tried with 2 different sd cards*

**Reply by Desuuuu** [reply]
Make sure you're using the proper firmware for your touchscreen. This kind of image glitch might indicate that the firmware is made for a different resolution screen or a different orientation.

**Reply by lanotlab** [reply]
I've downloaded it from the creality website, as well as the tiny-machines one, which came with the printer (the vendors sent me a copy), still no luck

**Reply by lanotlab** [reply]
It's the stock LCD screen

**Reply by lanotlab** [reply]
Ok, I "solved" it. Downloaded from four different mirrors and ONE of them worked just fine. All the files appear to be the same, but 3 of them give me this error and there's this one that worked just fine. I can now transit from marlin to klipper, with a functional screen, again. Thank you so much for the fork. Great work

**Comment by lanotlab** [comment]
https://drive.google.com/file/d/14_ENXpL1h15A2x000BmiRIsXx_W24a1M/view?usp=sharing

This is the firmware that worked, for anyone with the same problem
